### PR TITLE
85 login support quitting from login screen

### PR DESCRIPTION
**Pull request by jhulen**
Added ability to quit to login options

### CODE REVIEW

File: src/Login.cpp [hunk 1]
```diff
@@ -66,6 +66,11 @@ std::shared_ptr<User> Login::userInput() {
                 createUser(name, password);
                 break;
             }
+            case 'q':
+            case 'Q': {
+                printf("Exiting JimventoryManager.\n");
```
Comment (nshoap-utk) [inline src/Login.cpp:71]: This is the worst print statement I have ever seen.  REMOVE IT

**Review by nshoap-utk — CHANGES_REQUESTED** [review]

**Review by vbroda17 — DISMISSED** [review]

File: src/Login.cpp [hunk 2]
```diff
@@ -66,6 +66,11 @@ std::shared_ptr<User> Login::userInput() {
                 createUser(name, password);
                 break;
             }
+            case 'q':
+            case 'Q': {
+                printf("Exiting InventoryManager.\n");
```
Comment (nshoap-utk) [inline src/Login.cpp:71]: This is the best print statement I've ever seen -- great job!

Comment (jhulen) [reply src/Login.cpp:71]: <3

**Review by vbroda17 — APPROVED** [review]
Great Job. I miss the old print statement though.

**Pull request merged**